nntrainer/layers/layer_internal.h

```diff
@@ -312,7 +312,7 @@ class Layer : public ml::train::Layer {
    * @param method enum value to identify how it should be exported to
    */
   virtual void
-  export_to(const Exporter &exporter,
+  export_to(Exporter &exporter,
```

Comment:
this is just minor but how about `export` as function name?

Reply:
I tried but `export` is a reserved word, which makes ide complain about it. Let's leave this for now.
https://en.cppreference.com/w/cpp/keyword/export

Reply:
ohh, then this is fine.